src/components/footer/_macro-options.md

```diff
@@ -50,3 +50,19 @@
 | Name      | Type                                                        | Required | Description                 |
 | --------- | ----------------------------------------------------------- | -------- | --------------------------- |
 | itemsList | array`<ListItem>` [_(ref)_](/foundations/typography/#lists) | true     | A list of links for the row |
+
+## PoweredBy
+
+| Name               | Type                 | Required | Description                                                                             |
+| ------------------ | -------------------- | -------- | --------------------------------------------------------------------------------------- |
+| classes            | string               | false    | Classes to be added. Helpful to add a margin utility class to control spacing.          |
+| logo               | object`<LogoObject>` | false    | Logo object to render the poweredby logo                                                |
+| extraPoweredByLogo | object`<LogoObject>` | false    | Extra logo to go alongside the ONS logo in the footer                                   |
+| oppositeDisplay    | boolean              | false    | Enables an alternative display of the additional logo on the opposite side of the page. |
+
+## LogoObject
```

Comment:
Think this should just be named logo to be consistent with other documentation. Ive suggested the same for the header